client/src/components/questions-answers/SearchQuestions.jsx

```diff
@@ -1,29 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import SearchIcon from "@material-ui/icons/Search";
 import { useSelector, useDispatch } from 'react-redux';
-
+import IndividualQuestion from './individual-question/Index.jsx';
+import { Container, ListGroup, Row, Col } from 'react-bootstrap';
 
 
 const SearchQuestions = () => {
 
-  const answerList = useSelector(state => state.answerList);
+  const qnaList = useSelector(state => state.qnaList);
+  const searchBarTyped = useSelector(state => state.searchBarTyped);
   const dispatch = useDispatch();
-  const searchBarInput = useSelector(state => state.searchBarInput);
+  const [filteredData, setFilteredData] = useState([]);
+  const [ wordEntered, setWordEntered ] = useState('');
+
+  const handleSearch = (e) => {
+
+    e.preventDefault();
+    const searchWord = e.target.value;
+    setWordEntered(searchWord);
+
+    if (e.target.value.length !== 0) {
+      dispatch({
+        type:'SEARCHBAR_START_TYPING',
+        searchBarTyped: true
+      })
+    }
+
+    if (e.target.value.length === 0) {
+      dispatch({
+        type:'SEARCHBAR_START_TYPING',
+        searchBarTyped: false
+      })
+    }
+
+  const newFilter = qnaList.filter((value) => {
+    return value.question_body.toLowerCase().includes(searchWord.toLowerCase());
+  });
+
+
+
+  if (searchWord === "") {
+    setFilteredData([]);
+  } else {
+    setFilteredData(newFilter);
+  }
+}
+
+
+  let filteredqList = filteredData.map(question => {
+    return (
+     <ListGroup.Item key={question.question_id} >
 
-  // const handleFilter = (e) => {
-  //   const searchWord = e.target.input;
-  // }
+       <IndividualQuestion question={question} />
 
+     </ListGroup.Item>
+
+     )
+ })
 
 
   return (
 
-    <form className="form">
-      <input className="search-input" placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'/>
-      <SearchIcon />
-   </form>
+    <Container>
+
+      <Row>
+
+        <form className="form">
```

Comment:
As a heads up for future implementation, React-Bootstrap has a Form component that might make it easier to style! If this works for you though that's great, just wanted to make sure you know about it!

Reply:
yep, I used the react-bootstrap Form component in Modal, and just playing around with raw css. Thank you so much for heads up and all the help! :)